src/VisualStudio/Core/Def/DocumentOutline/DocumentOutlineControl_ModelComputation.cs

```diff
@@ -54,15 +56,26 @@ private void EnqueueComputeDataModelTask()
             if (filePath is null)
                 return null;
```

Comment:
here's an example of a problem.  you bail out early... but don't update hte progress bar visbiility.  see the below note about always running all parts of all of your queues.